practices/follow-functional-core-imperative-shell.md

```diff
@@ -3,117 +3,153 @@
 This ensures that the functional core remains isolated from external state changes, promoting clarity, predictability, and maintainability in the codebase while facilitating easier testing, debugging, and refactoring.
 
-### Learning Curve
+### Potentially Steep Learning Curve
 
-Transitioning to the Functional Core, Imperative Shell practice may present a learning curve.
+Transitioning to the Functional Core, Imperative Shell practice may present a steep learning curve for teams.
 To facilitate this transition smoothly, it's recommended that developers with more knowledge mentor other developers through pair programming sessions.
 Additionally, fostering an environment of knowledge sharing, providing resources, and allocating time for developers to study the practice can greatly aid in its adoption and understanding across the team.
 
 ### Is Incompatible with Object-Oriented Programming
 
-The Functional Core, Imperative Shell practice is not mutually exclusive with Object-Oriented Programming (OOP) principles.
-As long as the functional core remains free of side effects, it can coexist within an object-oriented architecture.
-The Functional Core's logic could be encapsulated within methods of a class, ensuring that these methods return new objects instead of mutating their fields.
-The Functional Core can take advantage of polymorphism by receiving or returning objects of a class hierarchy with multiple subclasses to represent various inputs or outputs.
-Developers can implement the functional core while adhering to OOP principles and have the advantages of both paradigms.
+<ed> I'm having trouble understanding the pitfall/challenge/limitation with this one. The text below implies developers can take advantage of both paradigms. Should we rephrase this subhead? Seems like "Can Be Compatible with OOP" is more in line with the text below. </ed> 
+
+The Functional Core, Imperative Shell practice and object-oriented programming (OOP) principles are not mutually exclusive.
+As long as the functional core remains free of side effects, it can co-exist within an object-oriented architecture.
+The functional core's logic could be encapsulated within methods of a class, ensuring that these methods return new objects instead of mutating their fields.
+The functional core can take advantage of polymorphism by receiving or returning objects of a class hierarchy with multiple subclasses to represent various inputs or outputs. <ed> I'm assuming your target reader will follow along with this explanation just fine. But this is one example of where this practice reads (to me) much more technical than others. What might help is relating this nuance back to code maintainability. </ed> 
+Developers can implement the functional core while adhering to OOP principles, taking advantage of both paradigms.
+
+### Imperative Shell Will Perform Various Tasks
```

Comment:
The essence of the point we were trying to make here is the following:

While it's best to remove ALL behavior/logic from imperative shells and have that instead live in functional cores, it's not always pragmatic to do so.

Reply:
This title doesn't convey that.

Reply:
What about: 
"Removing All Behavior or Logic From Imperative Shell is Ideal, but Impractical" 
"Some Behavior and Logic Needs to Live on Imperative Shell"

Reply:
Either is fine, thanks!

Reply:
Edited.